egs2/mucs21_subtask1/asr1/asr.sh

```diff
@@ -0,0 +1,1419 @@
+#!/usr/bin/env bash
```

Comment:
This file should not be presented in the egs2 but with a symlink from `TEMPLATE`. Could you refer to `egs2/TEMPLATE/setup.sh` for details?